src/app/components/api/overlayoptions.ts

```diff
@@ -6,6 +6,18 @@ import { AnimationEvent } from '@angular/animations';
  */
 export type OverlayModeType = 'modal' | 'overlay' | undefined;
 
+/**
+ * Represents the type of maximum width option, which can be a number, a number followed by 'px', or a number followed by 'vw'.
+ * @group Types
+ */
+export type MaxWidthOption = number | `${number}px` | `${number}vw`;
```

Comment:
Why not just use a string? And you can set any value you want with `%`, `rem`, etc.

Reply:
Because the popover element is not a child of the dropdown element. Using % would not scale it according to the control, but (probably) also to the whole document size. To not create a false hope, I went for only allowing vw, which should behave in the same way as %, while not confusing the user. 

If you want relative styling to the control width, you have to provide a number.